src/StudyApi/DependencyRegistrations/InfrastructureRegistration.cs

```diff
@@ -85,6 +87,12 @@ public static IServiceCollection AddInfrastructure(this IServiceCollection servi
             if (!ProdEnvironmentNames.Any(x => string.Equals(x, environmentName, StringComparison.OrdinalIgnoreCase)))
```

Comment:
I would change this config switch, any deployed environment won't be able to use these stubs not just prod.

Probably best to have an explicit config option to turn these on/off, you still might want to connect to the remote services locally too.